core/src/main/java/nu/yona/server/exceptions/ResourceBasedException.java

```diff
@@ -31,27 +33,17 @@ void onContextStarted(ContextRefreshedEvent event)
 	}
 
 	private static final long serialVersionUID = 8031973645020363969L;
-	/** Holds the parameters for the exception message. */
-	private final Object[] parameters;
-	/** Holds the message id. */
+	private final String[] messageArgs;
 	private final String messageId;
-	/** Holds the HTTP response code to be used. */
 	private final HttpStatus statusCode;
 	private static Translator translator;
 
-	/**
-	 * Constructor.
-	 * 
-	 * @param statusCode The status code of the exception.
-	 * @param messageId The ID of the exception in the resource bundle
-	 * @param parameters The parameters for the message
-	 */
-	protected ResourceBasedException(HttpStatus statusCode, String messageId, Object... parameters)
+	protected ResourceBasedException(HttpStatus statusCode, String messageId, Object... messageArgs)
 	{
 		super(messageId);
 
 		this.messageId = messageId;
-		this.parameters = parameters;
+		this.messageArgs = serializeMessageArgs(messageArgs);

```

Comment:
OK. I agree that this is a disadvantage. However, I think there is a good reason why exceptions should be serializable. Maybe a good solution is to format the message in the constructor and remove the messageArgs as a field?